pkgs/development/ocaml-modules/mirage-block-ramdisk/default.nix

```diff
@@ -10,6 +10,11 @@ buildDunePackage rec {
     sha256 = "cc0e814fd54efe7a5b7a8c5eb1c04e2dece751b7d8dee2d95908a0768896e8af";
   };
 
+  # Make compatible with cstruct 6.1.0
+  postPatch = ''
+    substituteInPlace src/ramdisk.ml --replace 'Cstruct.len' 'Cstruct.length'
+  '';
+
   minimalOCamlVersion = "4.06";
 
   propagatedBuildInputs = [ io-page mirage-block ];
```

Comment:
![image](https://user-images.githubusercontent.com/5861043/192358641-ffc4caae-d230-431e-91ae-9cbd978386a9.png)

> requires = "cstruct io-page lwt mirage-block"

Does this package work properly in downstream despite missing `cstruct` and `lwt` as propagated?